docs/source/tutorials/output_and_config.rst

```diff
@@ -276,25 +276,29 @@ When executing the command
 
    manim -pql scene.py SquareToCircle
 
-it was necessary to specify which ``Scene`` class to render.  This is because a
-single file can contain more than one ``Scene`` class.  If your file contains
-multiple ``Scene`` classes, and you want to render them all, you can use the
-``-a`` flag.
-
-As discussed previously, the ``-ql`` specifies low render quality.  This does
-not look very good, but is very useful for rapid prototyping and testing.  The
-other options that specify render quality are ``-qm``, ``-qh``, and ``-qk`` for
-medium, high, and 4k quality, respectively.
+it specifies the scene to render.  This is not necessary now.  When a single
+file contains only one ``Scene`` class, it just render the only one. When a
```

Comment:
```suggestion
file contains only one ``Scene`` class, it will just render the ``Scene`` class. When a
```
Seems your force push broke your previous commit?

Reply:
There was a typo in previous commit message, so I push -f to fix it. Sorry for that😪